src/stylesheets/components/_user_card.scss

```diff
@@ -1,10 +1,42 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:400,900');
+
 .user_card {
   display: flex;
+  flex-direction: column;
+  justify-content: space-between;
   align-items: center;
-  text-align: left;
-  img {
-    margin-right: 20px;
-    width: 70px;
-    height: 70px;
+  text-align: center;
+  position: relative;
+  height: 250px;
+  width: 220px;
+  bottom: 0px;
+  font-family: 'Lato', 'sans-serif';
```

Comment:
So if we're thinking of using Lato at any other point in code, maybe we can make a SASS variable out of it-- that way if we decide later we need to change the font, we're only changing it in one place